script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -131,7 +132,8 @@ I won't feel any pain.
 }
 [sha:ced3fa1d42582b23cd0e4fb697a1cde6c2773b5f]{
 -- Page 164, Offset 29973.
--- だからこんな時間、幸福で息がつまる。
+-- だからこんな時間、幸福で息がつまる。\
+// group pr check: should be the same to the prev line
```

Comment:
I feel like it's probably clear this is a callback, I don't think we need to force same phrasing

Reply:
i agree